### PR TITLE
add support to rails 7

### PR DESCRIPTION
**Pull request by roman-melnyk**

### CODE REVIEW

**Comment by roman-melnyk** [comment]
@jcoleman hey! could you please look what's wrong with specs, they passed locally, but failed in semaphoreci , and I do not see details

File: spec/active_record_helper/setup.rb [hunk 1]
```diff
@@ -14,7 +14,12 @@
 }
 
 env = ENV['DATABASE'] ||= 'sqlite'
-config = ActiveRecord::Base.configurations[env]
+config =
+  if ActiveRecord::Base.configurations.respond_to?('[]')
```
Comment (jcoleman) [inline spec/active_record_helper/setup.rb]:
```suggestion
  if ActiveRecord::VERSION::MAJOR < 6
```

File: Appraisals [hunk 2]
```diff
@@ -1,7 +1,7 @@
-%w(5.0 5.1 5.2 6.0).each do |version|
+%w(5.0 5.1 5.2 6.0 6.1 7.0).each do |version|
   appraise "rails-#{version.gsub(/\./, "-")}" do
     gem "rails", "~> #{version}.0"
-    if version[0] == '6'
+    if version[0].to_i >= 6
```
Comment (jcoleman) [inline Appraisals]:
```suggestion
    if Gem::Version.new(version) >= Gem::Version.new("6.0.0")
```

**Review by jcoleman — COMMENTED** [review]
Looks mostly good to me; would you apply the suggested changes and see if it still passes CI?

File: Appraisals (hunk 2 above)
Comment (roman-melnyk) [reply Appraisals]: done

**Pull request merged**